### PR TITLE
app-portage/pkg-testing-tools: add 0.2.0

### PR DESCRIPTION
**Pull request by APN-Pucky**
Original repo is in archive mode now https://github.com/slashbeast/pkg-testing-tools and read only https://github.com/slashbeast/pkg-testing-tools/issues/16#issuecomment-1829574242

I'd keep a fork with patches/PRs alive.
Above issue is already fixed in my repo per https://github.com/APN-Pucky/pkg-testing-tools/commit/0666ce82fea1a55c95367a39d0a65dba6f8277bd.
Maybe @slashbeast can/should approve of this transition or should this happen over gentoo's bugzilla?

Signed-off-by: Alexander Puck Neuwirth <alexander@neuwirth-informatik.de>

### CODE REVIEW

File: app-portage/pkg-testing-tools/metadata.xml [hunk 1]
```diff
@@ -2,6 +2,10 @@
 <!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
 	<!-- maintainer-needed -->
+	<maintainer type="person" proxied="yes">
+		<email>alexander@neuwirth-informatik.de</email>
+		<name>Alexander Puck Neuwirth</name>
+	</maintainer>
```
Comment (mm1ke) [inline app-portage/pkg-testing-tools/metadata.xml:8]: You need to add
```
	<maintainer type="project" proxied="proxy">
		<email>proxy-maint@gentoo.org</email>
		<name>Proxy Maintainers</name>
	</maintainer>
```
in order to make CI happy

Comment (APN-Pucky) [reply app-portage/pkg-testing-tools/metadata.xml:8]: Thanks, I changed that and split the metadata changes in a separate commit.

**Review by juippis — APPROVED** [review]
Awesome! I'm a heavy user of this tool so glad to see it's able to live on. Could you look into adding python-3.12 support and removing 3.9 while at it?

Feel free to ping me directly on pkg-testing-tool issues.

**Comment by APN-Pucky** [comment]
I'll check the python3.12 support. I use it also a lot (even to test itself :) https://gitlab.com/APN-Pucky/gentoo-apn/-/merge_requests/55/pipelines, here I test ~riscv and ~arm64) 
The other issue I want to solve is the handling of `SINGLE_TARGET_*` cases.

**Comment by juippis** [comment]
I've just added it to be ignored, 
https://github.com/juippis/my-gentoo-lxd-scripts/blob/master/pkg-testing-tool-patches/mtc/0.1.2/pkg-testing-tools-0.1.2-toggle-more-flags-generic.patch
still selected from profile. It is trouble when something ONLY supports single implementation that's newer than the profile default. This **very** rarely happens.

**Comment by APN-Pucky** [comment]
Currently I use something like this
`pkg-testing-tools ... --append-required-use "^^ ( python_single_target_python3_11 python_single_target_python3_10 ) python_single_target_$PY`,
where $PY is looped over.
Maybe that can be integrated to work more reliably for any set of supported pythons, we'll see.

**Pull request closed without merging**